pkg/aws/util.go

```diff
@@ -45,6 +48,15 @@ func (d *Driver) createSVC(secret *corev1.Secret, region string) (ec2iface.EC2AP
 	return svc, nil
 }
 
+// Function returns true only if error code equals codes.NotFound
+func isNotFoundError(err error) bool {
```

Comment:
can you check places where we extract the error code and compare it against `codes.NotFound` and in those places use this function instead?
One place on quick search is `core_util.go/terminateInstance`

Reply:
We cannot change `terminateInstance` because here we extract the MCM error code from the provider error and then compare it with `codes.NotFound`. We have used this function in one more place, where we found the usage was possible.